Bump wg-access-server release to latest

awlx opened the pull request:
https://github.com/freifunkMUC/wg-access-server/releases/tag/v0.9.0

awlx committed:
Bump wg-access-server release to latest

https://github.com/freifunkMUC/wg-access-server/releases/tag/v0.9.0

nwaldispuehl commented:
Looks fine!

I also wanted to add these new DNS config client settings from #17 .
Can/should I add it to this PR?

awlx commented:
Yep, just feel free to push to this one :).

nwaldispuehl committed:
Adds hints towards the new custom client config DNS setting to the default `values.yaml` file.

Also adds a link to the wg-access-server config documentation to the `README.md` file.

Closes #17.
